tests/fixed/ClassPropertySpacing.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Spacing;
+
+final class ClassPropertySpacing
+{
+    public bool $foo;
+
+    /** @var string[] */
```

Comment:
IMO not necessary to have added spacing here: the minimum can stay zero

Reply:
I'm like "I don't care and don't know on this". I like both (olol), so I kept sniff's default.

What do others think?

Reply:
I would allow both 0 and 1